draft-ietf-rtcweb-jsep.xml

```diff
@@ -1859,6 +1859,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             This option allows the endpoint to significantly reduce the amount of
             audio bandwidth it receives, at the cost of some fidelity, depending on
             the quality of the remote VAD algorithm.</t>
+            <t>If the "VoiceActivityDetection" option is specified, with a value
+            of "false", the browser MUST negotiate the audio to not suppress any
```

Comment:
Would prefer this to mirror the section above, e.g.

"the offer MUST indicate that it does not want to use silence suppression, by not including any comfort noise codecs and by setting any codec-specific settings appropriately, e.g. usedtx=0 for Opus."